app/src/main/java/com/worldsnas/starplayer/view/musics_list/MusicListViewModel.kt

```diff
@@ -31,4 +31,17 @@ class MusicListViewModel @Inject constructor(
     fun postMusic(): LiveData<List<MusicRepoModel>> {
         return postMusicList
     }
+
+    private fun saveMusic() {
+        viewModelScope.launch {
+            musicRepository.saveToDatabase()
```

Comment:
in kar che rabti ViewModel dare?
shoma bayad in karo khodeton toye repo anjam bedid 
bedone inke laye haye bala befahman